File: resourcespoet/src/test/java/com/commit451/resourcespoet/PluralsTest.java
package com.commit451.resourcespoet;

import org.junit.Test;

import java.util.ArrayList;

/**
 * Tests the resources creation
 */
public class PluralsTest {

    @Test
    public void pluralsTest() throws Exception {
        ArrayList<Plural> plurals = new ArrayList<>();
        plurals.add(new Plural(Plural.Quantity.one, "%d song"));
        plurals.add(new Plural(Plural.Quantity.other, "%d songs"));

        ResourcesPoet poet = ResourcesPoet.create()
                .addPlurals("songs", plurals);

        TestUtil.assertEquals("plurals.xml", poet);
    }
}

File: src/test/kotlin/com/commit451/resourcespoet/PluralsTest.kt
package com.commit451.resourcespoet

import org.junit.Test

/**
 * Tests the resources creation
 */
class PluralsTest {

    @Test
    fun pluralsTest() {
        val plurals = ArrayList<Plural>()
        plurals.add(Plural(Plural.Quantity.one, "%d song"))
        plurals.add(Plural(Plural.Quantity.other, "%d songs"))

        val poet = ResourcesPoet.create()
            .addPlurals("songs", plurals)

        TestUtil.assertEquals("plurals.xml", poet)
    }
}
